common/util/file_util.cc

```diff
@@ -290,6 +290,9 @@ absl::StatusOr<Directory> ListDir(absl::string_view dir) {
   return d;
 }
 
+static const char *STDIN_FILENAME = "-";
+bool IsStdin(absl::string_view filename) { return filename == STDIN_FILENAME; }
```

Comment:
Can you also use this new function in the `GetContentAsString()` function, around line 155 ?

Reply:
Sure! Didn't notice that part of the code. This answers my doubt about stdin being uniquely identified by "-". Applied suggested changes too.